src/Logging.Legacy/OrleansLoggingUtils.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Orleans.Runtime;
+using System;
+using System.Net;
+using System.Threading;
+
+namespace Microsoft.Orleans.Logging.Legacy
+{
+    /// <summary>
+    /// Utility method for OrleansLogging
+    /// </summary>
+    public class OrleansLoggingUtils
+    {
+        internal const string ObsoleteMessageString =
+                "The Microsoft.Orleans.Logging.Legacy namespace was kept to facilitate migration from Orleans 1.x but will be removed in the near future. It is recommended that you use the Microsoft.Extensions.Logging infrastructure and providers directly instead of Microsoft.Orleans.Logging.Legacy.Logger and Microsoft.Orleans.Logging.Legacy.ILogConsumer";
```

Comment:
nit: it's not `Microsoft.Orleans.Logging.Legacy.Logger` any more, it's `Microsoft.Orleans.Logging.Legacy.OrleansLegacyLogger`.

Reply:
? Microsoft.Orleans.Logging.Legacy.Logger is the namespace, which is what we want to refer here right?

Reply:
no I think we are referring to our old Orleans.Runtime.Logger abstraction here. and in the future, Orleans.Runtime.Logger will be changing to Microsoft.Orleans.Logging.Legacy.Logger

Reply:
ahh, ok, I remember now :)